training/finetune_acft_hf_nlp/components/pipeline_components/translation/spec.yaml

```diff
@@ -523,7 +523,7 @@ jobs:
       mlflow_model: ${{parent.jobs.translation_finetune.outputs.mlflow_model_folder}}
   compute_metrics:
     type: command
-    component: azureml://registries/azureml-preview/components/compute_metrics/versions/0.0.9.preview
+    component: azureml://registries/azureml/components/compute_metrics/versions/0.0.1
```

Comment:
can we start with v1 in a public registry?

Reply:
@vizhur you mean version to be set to 1.0.0? We already published command components with 0.0.1 and hence maintaining the same version.

Reply:
cc: @imbalu007

Reply:
@vizhur, the components are still in public preview. At GA we plan to go with v1

Reply:
sure, 0.0.1 doesn't look good for PuP either